src/api/network.js

```diff
@@ -2,7 +2,8 @@ const { getInstance } = require('../qclient');
 
 const Network = {
   async getPeerNodeCount() {
-    return getInstance().getPeerInfo().length || 0;
```

Comment:
`return await (getInstance().getPeerInfo()).length || 0;`

you need to await `getPeerInfo` since that is returning a `Promise` before you try to access the data (length).